Assets/Scripts/LoadNextLevel.cs

```diff
@@ -0,0 +1,35 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class LoadNextLevel : MonoBehaviour
+{
+    public int nextSceneLoad;
+    // Start is called before the first frame update
+    void Start()
+    {
+        nextSceneLoad = SceneManager.GetActiveScene().buildIndex + 1;
+    }
+
```

Comment:
What's exactly the purpose of this script? I think we should discuss that.

Reply:
Script has been deleted from the project.